yt_dlp/extractor/kick.py

```diff
@@ -117,12 +143,100 @@ def _real_extract(self, url):
             'uploader': traverse_obj(response, ('livestream', 'channel', 'user', 'username')),
-            'uploader_id': str_or_none(traverse_obj(response, ('livestream', 'channel', 'user_id'))),
+            'uploader_id': int_or_none(traverse_obj(response, ('livestream', 'channel', 'user_id'))),
             'timestamp': unified_timestamp(response.get('created_at')),
             'duration': float_or_none(traverse_obj(response, ('livestream', 'duration')), scale=1000),
             'thumbnail': traverse_obj(
                 response, ('livestream', 'thumbnail'), expected_type=url_or_none),
             'categories': traverse_obj(response, ('livestream', 'categories', ..., 'name')),
         }
+
+
+class KickClipIE(KickBaseIE):
+    IE_NAME = 'kick:clips'
+    _VALID_URL = r'https?://(?:www\.)?kick\.com/(?P<channel>[\w-]+)\?clip=(?P<id>clip_[\w-]+)'
+    _TESTS = [{
+        'url': 'https://kick.com/mxddy?clip=clip_01GYXVB5Y8PWAPWCWMSBCFB05X',
+        'info_dict': {
+            'id': 'clip_01GYXVB5Y8PWAPWCWMSBCFB05X',
+            'ext': 'mp4',
+            'livestream_id': 1855384,
+            'title': 'Maddy detains Abd D:',
+            'category_id': 64,
+            'channel': 'Mxddy',
+            'channel_id': 133789,
+            'uploader': 'AbdCreates',
+            'uploader_id': 3309077,
+            'thumbnail': r're:^https?://.*\.jpeg',
+            'duration': 35,
+            'category': 'VALORANT',
+            'upload_date': '20230426',
+            'view_count': int,
+            'like_count': int,
+            'is_mature': True,
+        },
+        'params': {
+            'skip_download': 'm3u8',
+        },
+        'expected_warnings': [r'impersonation'],
+    }, {
+        'url': 'https://kick.com/destiny?clip=clip_01H9SKET879NE7N9RJRRDS98J3',
+
+        'info_dict': {
+            'id': 'clip_01H9SKET879NE7N9RJRRDS98J3',
+            'title': 'W jews',
+            'ext': 'mp4',
+            'view_count': int,
+            'like_count': int,
+            'is_mature': False,
+            'duration': 49.0,
+            'thumbnail': 'https://clips.kick.com/clips/j3/clip_01H9SKET879NE7N9RJRRDS98J3/thumbnail.png',
+            'uploader_id': 2027722,
+            'channel': 'Destiny',
+            'channel_id': 1772249,
+            'category_id': 15,
+            'uploader': 'punished_furry',
+            'category': 'Just Chatting',
+            'livestream_id': 14549749,
+            'upload_date': '20230908',
+        },
+        'params': {
+            'skip_download': 'm3u8',
+        },
+        'expected_warnings': [r'impersonation'],
+
+    }]
+
+    def _real_extract(self, url):
+
+        channel_slug, clip_id = self._match_valid_url(url).groups()
+        response = self._call_api(f'v2/clips/{clip_id}/play', clip_id, note='Getting clip information')
+        clip = traverse_obj(response, 'clip', expected_type=dict)
+        clip_url = traverse_obj(clip, 'clip_url')
```

Comment:
same here, it's ok for `clip_url` to be fatal
```suggestion
        clip_url = clip['clip_url']
```